### ISSUE TITLE
Add implementations of LR schedulers

### ISSUE THREAD

**Issue by LovePelmeni** [LovePelmeni/DeepfakeStream#6]
Implement learning rate schedulers: PolyLR, StepLR, ExponencialLR, inherited from the base scheduler torch.optim.lr_scheduler._LRScheduler.
